features/foutafhandeling.feature

```diff
@@ -0,0 +1,542 @@
+        En bevat de response de volgende invalidParams
+        | name   | code   | reason                                                                     |
+        | expand | expand | Deel van de parameterwaarde is niet correct: nummeraanduiding.bestaatniet. |
+
+Rule: embedden van een gerelateerde resource i.c.m. de gerelateerde resource identificatie query parameter is niet toegestaan
+
+    Abstract Scenario: filteren van resources met een gerelateerde resource identificatie i.c.m. het embedden van dezelfde gerelateerde resource
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        En bevat de response de volgende invalidParams
+        | name   | code   | reason   |
+        | expand | <code> | <reason> |
+
+        Voorbeelden:
+        | path                   | query string                                                                  | code   | reason                                                                                                                                                      |
+        | /adresseerbareobjecten | ?pandIdentificatie=0226100000008856&expand=panden                             | expand | Deel van de parameterwaarde is niet correct: filteren met pandIdentificatie en tegelijk expanden van panden is niet toegestaan.                             |
+        | /adressen              | ?adresseerbaarObjectIdentificatie=0226010000038820&expand=adresseerbaarObject | expand | Deel van de parameterwaarde is niet correct: filteren met adresseerbaarObjectIdentificatie en tegelijk expanden van adresseerbaarObject is niet toegestaan. |    
+
+Rule: wanneer een parameter geometrie bevat in een crs anders dan RD (epsg:28992) wordt een 400 fout gegeven
+
+    Abstract Scenario: accept-crs WGS84 geeft een foutmelding
+        Als '<path><query string>' request wordt gedaan met header 'Accept-Crs: epsg:4326'
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                                              |
+        | title  | Coördinatenstelsel epsg:4326 in Content-Crs wordt niet ondersteund. |
+        | status | 400                                                                 |
+        | code   | crsNotSupported                                                     |
+        En bevat de response geen veld "invalidParams"
+        En is de http status code van het antwoord 400
+        En bevat de response de header "Content-Type" met waarde "application/problem+json"
+
+        Voorbeelden:
+        | path                   | query string                            |
```

Comment:
Voor deze voorbeelden krijg ik een andere response

/panden?locatie=98095,438495
```
{
    "status": 406,
    "title": "Gevraagde coördinatenstelsel epsg:4326 wordt niet ondersteund.",
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.6",
    "detail": "Not Acceptable.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/panden?locatie=98095,438495",
    "code": "crsNotAcceptable"
}
```

/panden?locatie=51.93201,4.55980
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "Een of meerdere parameters zijn niet correct.",
    "status": 400,
    "detail": "Bad request.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/panden?locatie=51.93201,4.55980",
    "code": "paramsValidation",
    "invalidParams": [
        {
            "name": "locatie",
            "code": "geometryMismatch",
            "reason": "Waarde is niet conform opgegeven CRS."
        }
    ]
}
```

/panden?bbox=135228,457502,135246,457457
```
{
    "status": 406,
    "title": "Gevraagde coördinatenstelsel epsg:4326 wordt niet ondersteund.",
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.6",
    "detail": "Not Acceptable.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/panden?bbox=135228,457502,135246,457457",
    "code": "crsNotAcceptable"
}
```

/adresseerbareobjecten?bbox=135228,457502,135246,457457
```
{
    "status": 406,
    "title": "Gevraagde coördinatenstelsel epsg:4326 wordt niet ondersteund.",
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.6",
    "detail": "Not Acceptable.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/adresseerbareobjecten?bbox=135228,457502,135246,457457",
    "code": "crsNotAcceptable"
}
```

/adresseerbareobjecten?bbox=52.10540,5.09861,52.10500,5.09887
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "Een of meerdere parameters zijn niet correct.",
    "status": 400,
    "detail": "Bad request.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/adresseerbareobjecten?bbox=52.10540,5.09861,52.10500,5.09887",
    "code": "paramsValidation",
    "invalidParams": [
        {
            "name": "bbox",
            "code": "geometryMismatch",
            "reason": "Waarde is niet conform opgegeven CRS."
        }
    ]
}
```

Reply:
In de Als werd de verkeerde header aangegeven, moet zijn Content-Crs